### ISSUE TITLE
`[SceneReferenceOptions]` attribute does not work for nested fields

### ISSUE THREAD

**Issue by starikcetin** [starikcetin/Eflatun.SceneReference#18]
This is an example of when it doesn't work:

```cs
public class NestedOptionsTest : MonoBehaviour
{
    [SerializeField] private Container container;
}

[Serializable]
public class Container
{
    [SceneReferenceOptions(UtilityLine = UtilityLineBehaviour.Disabled, Coloring = ColoringBehaviour.Enabled)] 
    [SerializeField] private SceneReference sceneReferenceField;
}
```

**Comment by starikcetin** [comment]
Fixed in #19. Will be included in the next release.

**Comment by starikcetin** [comment]
Included in [`1.2.2`](https://github.com/starikcetin/Eflatun.SceneReference/releases/tag/1.2.2).
